### DISCUSSION TITLE
Need Select Box component

### DISCUSSION THREAD

**Discussion by abdulmateentechbits** [infinitered/ignite#2366] [Q&A] [answered]
Any one please share link for select box input custom component like we have for TextField

**Comment by jamonholmgren** [comment, marked as answer]
Here ya go -- @yulolimum made one just for you!

https://ignitecookbook.com/docs/recipes/SelectFieldWithBottomSheet/

**Reply by abdulmateentechbits** [reply]
Thanks sir 🫂

**Reply by abdulmateentechbits** [reply]
> Here ya go -- @yulolimum made one just for you!
> 
> https://ignitecookbook.com/docs/recipes/SelectFieldWithBottomSheet/

Sir how can i show selected while on update the field ?
